src/edge/blast.py

```diff
@@ -84,14 +85,14 @@ def blast(dbname, blast_program, query):
         infile = f.name
         f.write(">Query\n%s\n" % query)
 
-    outfile = "%s.out.xml" % infile
+    outfile = "%s.out.json" % infile
     if blast_program == "tblastn":
         blast_cl = NcbitblastnCommandline(
-            query=infile, db=dbname, word_size=6, outfmt=5, out=outfile
+            query=infile, db=dbname, word_size=6, outfmt=15, out=outfile, num_threads=4
         )
     else:
         blast_cl = NcbiblastnCommandline(
-            query=infile, db=dbname, word_size=6, outfmt=5, out=outfile
+            query=infile, db=dbname, word_size=6, outfmt=15, out=outfile, num_threads=4
```

Comment:
Can we have "4" be instead an env variable, that if is not defined or defined to null, then defaults to 4? This way, in Rancher, we can deploy with a higher number if we got a beefier machine.